storyruntime/Exceptions.py

```diff
@@ -15,7 +15,11 @@ def __str__(self):
             return super().__str__()
 
         stack = self.story.get_stack()
-        trace = f'An exception has occurred:\n{self.message}'
+
+        trimmed_msg = f'{self.message[0:125]}...' if \
+            len(self.message) >= 125 else self.message
```

Comment:
Broken for n == 125.
How about:

```suggestion
        trimmed_msg =  self.message[:MAX_LOG_MESSAGE_LENGTH]
        if len(self.message) > MAX_LOG_MESSAGE_LENGTH:
            trimmed_msg += '...'
```

Reply:
@TonyRice looks good. Let's apply it, add a small test, and merge?